packages/babel-preset-react-app/index.js

```diff
@@ -30,7 +30,11 @@ const plugins = [
     regenerator: true,
     // Resolve the Babel runtime relative to the config.
     moduleName: path.dirname(require.resolve('babel-runtime/package'))
-  }]
+  }],
+  // We always include this plugin regardless of environment
+  // because of a Babel bug that breaks object rest/spread without it:
+  // https://github.com/babel/babel/issues/4851
+  require.resolve('babel-plugin-transform-es2015-parameters'),
```

Comment:
I think this [was fixed](https://github.com/babel/babel/pull/5088) in `babel@6.22.0`.